src/main/java/org/commonjava/util/gateway/services/ProxyConstants.java

```diff
@@ -1,7 +1,14 @@
 package org.commonjava.util.gateway.services;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class ProxyConstants
 {
     // Vert.x event types
     public static final String EVENT_PROXY_CONFIG_CHANGE = "proxy-config-change";
+
+    // Auto generated, ignore such upstream headers
+    public static final List<String> FORBIDDEN_HEADERS =
```

Comment:
Maybe Set is better here as its contains method is faster than ArrayList.

Reply:
opps. I missed this. anyway, let's test it first.